conftest.py

```diff
@@ -63,7 +63,21 @@ def pytest_runtest_setup(item):
 )
 
 
-@pytest.fixture(params=["disk", "tasks"])
+@pytest.fixture(
```

Comment:
Copied from https://github.com/dask/dask/pull/8250#issuecomment-944719188

I'm not sure we'll want exactly this in the end. I think we may want something that can be controlled with a command-line flag to pytest (https://stackoverflow.com/a/63271994/17100540)? That way:
* In dask/dask, shuffle tests would not use `p2p`
* In dask/distributed, within `distributed/shuffle/tests/test_dask.py`, we can `from dask.dataframe.tests import test_shuffle`—this will cause the dask tests to run within distributed's CI
* In dask/distributed, we invoke pytest with the `--shuffle-p2p` pytest flag we set up, so the imported tests from dask only use the `p2p` shuffle fixture.